lib/languages/js/index.js

```diff
@@ -56,6 +58,9 @@ function analyzeScope(ast, languageOptions, visitorKeys) {
 // Exports
 //-----------------------------------------------------------------------------
 
+/**
+ * @implements {Language}
```

Comment:
```suggestion
 * @type {Language}
```

I believe `@implements` is for classes. And intellisense seems to be working well when this is changed to `@type`.